spec/features/appropriate_bodies/appropriate_bodies_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
do we want to add a similar feature test for choosing roles, or is this covered by other specific feature tests?